src/components/Modal/__stories__/modal.stories.mdx

```diff
@@ -379,6 +400,17 @@ clicking outside. Use this variant in case of sensitive or important messages, a
   ]}
 />
 
+<Tip title="Devs tip">
+  If you wish to implement a dropdown inside a Modal container and need help displaying the dropdown's popovers
+  correctly, read more about our dropdown in popovers technical pattern
+  <Link
+    href="?path=/docs/technical-patterns-dropdowns-inside-pop-overs--modal-with-damaged-dropdown"
```

Comment:
Link is not working correctly. I think `href` should start with `/?`, but check it, please